graphium/ipu/ipu_losses.py

```diff
@@ -1,11 +1,56 @@
 import torch
 from torch import Tensor
-from torch.nn import BCELoss, MSELoss, L1Loss
+from torch.nn import BCELoss, BCEWithLogitsLoss, MSELoss, L1Loss
 from torch._C import _infer_size
 from loguru import logger
 from graphium.trainer.losses import HybridCELoss
 
 
+class BCEWithLogitsLossIPU(BCEWithLogitsLoss):
```

Comment:
@DomInvivo Are you happy with this code duplication? I could try to refactor it.